app/src/main/java/com/example/supercookie/ChargeCall.java

```diff
@@ -1,7 +1,9 @@
 package com.example.supercookie;
 
-import android.support.annotation.NonNull;
 import android.util.Log;
+
+import androidx.annotation.NonNull;
+
```

Comment:
Run "Optimize imports" (ctrl+option+O), it will remove the unnecessary lines

Reply:
Actually, it added more, between what it thought were blocks of domain ownership, I think?  But I removed them....